DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -24,17 +24,17 @@ public void Setup()
         public void Get_course_content_should_return_course()
         {
             // When
-            const int customisationId = 1;
+            const int customisationId = 1604;
             var result = courseContentService.GetCourseContent(customisationId);
 
             // Then
             var expectedCourse = new CourseContent(
-                1,
-                "Entry Level - Win XP, Office 2003/07 OLD",
-                "Standard",
-                "N/A",
-                "North West Boroughs Healthcare NHS Foundation Trust",
-                "xxxxxxxxxxxxxxxxxxxx"
+                1604,
+                "Level 2 - Microsoft Word 2007",
+                "Word Core 07 Testing",
+                "3h 56m",

```

Comment:
Nice, good to know this is the format 🙂 Also @SChandarana worth noting that it can be N/A. Worth checking what the old system displays in that case (whether it displays N/A or nothing) and doing the same 🙂

Reply:
The old system displays the duration in the same format. I've not managed to find an example where the duration is "N/A", but given that the underlying SQL function is just returning a string of either that format, or "N/A", I suspect the duration displayed by the old website will be "N/A" for courses without a duration. 

A major part of the SQL function is formatting, as well producing an average duration in minutes to format. I wonder if it would be nicer to bring that closer to the front end and instead have this query just return a nullable number of minutes. What do you think? 🙂

Reply:
We could do but I'd be tempted to leave it like this, at least if we don't have any issues with it. If we can reuse something like this from the old system without needing to modify it all then I think it's worth doing 🙂

Reply:
(I mainly meant to as a comment for Shiv to bear in mind when implementing the front end, sorry if that wasn't clear 🙂 )

Reply:
Ah I see. That makes sense 🙂